practices/smart_goals.md

```diff
@@ -0,0 +1,123 @@
+In order to set a relevant goal, answer these questions (answers should be “Yes”): 
+
+- Does this seem worthwhile?
+- Is this the right time?
+- Does this match my other efforts?
+- Am I the right person to reach this goal?
+- Is it applicable in the current (socio-)economic environment?
+
+### Time-bound
+
+What is your goal time-frame? An end-date can provide motivation and help you to prioritize and prevent everyday tasks from taking priority over your longer-term goals.
+To set a time-bound goal, you will deal with the following questions:
+
+- When?
+- What can I do six months from now?
+- What can I do six weeks from now?
+- What can I do today?
+
+## How to Start With SMART Goals
+
+There are various SMART Goals worksheets, templates, and maps that can help you with the goals setting process. 
+
+![](/files/smart_templ1b.png)
+
+## Problems That SMART Goals Can Solve
+
+SMART Goals can be used for problems that you are able to define fully following SMART Goals methodology carefully. 
+
+There are various goals that SMART Goals cannot solve: 
+
+- goals that do not allow definition of a clear result
+- goals whose progress cannot be measured
+- goals whose achieving depends on circumstances that you cannot affect
+- etc.
+
+For example, it will be very difficult (or impossible) to define a clear result and relevant metrics for problems like __**Not my problem mentality**__.
+
+## SMART Goals Pitfalls 
+
+Significant problems of SMART goal setting:
+
+- Focus is too narrow – fixating on a single SMART goal without the appropriate context, it is easy to fall into the trap of seeing it as the only goal.
+- Measuring success and failure – when success is measured by a SMART goal, people may end up pursuing that goal for a narrowly defined success, and letting that exclusive target take over their identity.
+- Short-term result vs. long-term success – preferring short-term results to overall success can represent a risk of missing your goal. 
+- Giving up too soon, and the all-or-nothing approach – SMART goals are viewed as complete entities, and when you are unable to do everything as planned, you may become discouraged and give up the entire goal.
+- Failing to realize one's full potential – setting goals that are too easy will not move you to achieve more than your minimum potential; so could you miss the opportunity for growth.
+- “Realistic” and “Achievable” can be misleading – ask yourself not only whether your goal is realistic considering your capability but also whether it is realistic considering all your other goals. Do not overload yourself with too many high priorities.
+- Missing a crucial step along the way – be aware of setting a clear first step and final result but not knowing how you are going to get from A to B.
+
+## Resources for SMART Goals
+
```

Comment:
I like that you went for your own sources, good job!

Reply:
Thanks